molecules/sample_request.json

```diff
@@ -0,0 +1 @@
+{"TotalC": 9, "TotalH": 17, "TotalO": 4, "TotalN": 1}
```

Comment:
this seems to be the processed format.  does the deployed pipeline include also the preprocessing steps?